pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -490,6 +490,69 @@ var testsRequiringHistoryRewrite = map[testCoordinates]string{
 		jobName:  "periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-upgrade",
 		testName: "[sig-api-machinery] Kubernetes APIs remain available with reused connections",
 	}: "history correction on improperly loose disruption criteria, expires 2022-01-15",
+
+	{
+		jobName:  "aggregated-azure-ovn-upgrade-4.10-micro-release-openshift-release-analysis-aggregator",
+		testName: "[sig-arch][bz-kube-apiserver][Late] Alerts alert/KubeAPIErrorBudgetBurn should not at or above info [Suite:openshift/conformance/parallel]",
+	}: "history correction on improperly loose disruption criteria, expires 2022-01-24",
+	{
+		jobName:  "aggregated-azure-ovn-upgrade-4.10-micro-release-openshift-release-analysis-aggregator",
+		testName: "[bz-kube-apiserver][invariant] alert/KubeAPIErrorBudgetBurn should not at or above info",
```

Comment:
deads just changed this to "should not be at" in https://github.com/openshift/origin/pull/26743